### DISCUSSION TITLE
Quick question, is lingui's t macro same as i18nMark? Did the i18nMark function get deprecated? It's not mentioned in the migration guide.

### DISCUSSION THREAD

**Discussion by shibisuriya** [lingui/js-lingui#1795] [Q&A] [answered]
**Describe the bug**

**To Reproduce**
Steps to reproduce the behavior, possibly with minimal code sample, e.g:

```tsx
import { Trans } from "@lingui/react"

export default function App() {
   return <Trans>This should be translated!</Trans>
}
```

**Expected behavior**

**Additional context**

- jsLingui version ``lingui --version``
- Babel version ``npm list @babel/core``
- Macro support:
 - [ ] I'm using SWC with `@lingui/swc-plugin`
 - [ ] I'm using Babel with `babel-macro-plugin`
 - [ ] I'm not using macro
- Your Babel config (e.g. `.babelrc`) or framework you use (Create React App, NextJs, Vite)

**Comment by andrii-bodnar** [comment]
The `i18nMark` function was removed in the Lingui v3 release

**Comment by shibisuriya** [comment]
Btw what does it do @andrii-bodnar? I can't see what it does in old docs (since there are no old docs), I want to know what it does since I want to migrate to latest version of lingui.

**Comment by andrii-bodnar** [comment, marked as answer]
@shibisuriya here is the old documentation for this function - https://github.com/lingui/js-lingui/blob/stable-2.x/docs/ref/react.rst#i18nmark

In Lingui v4, you can use the [`msg`/`defineMessage`](https://lingui.dev/ref/macro#definemessage) as an alternative
